config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  namespace :api, defaults: { format: 'json' } do
+    get '/messages', to: 'messages#index'
+  end
+
+  root 'static#index'
```

Comment:
- [ ] Something is wrong in your route or `fetchmessage` function in `redux/reducers/messages` file. Please fix it.

![image](https://user-images.githubusercontent.com/38635158/159363904-c241c09d-465d-4c39-9cbb-7ac9d231790f.png)